src/timing.h

```diff
@@ -392,6 +406,44 @@ struct jl_timing_suspend_cpp_t {
     _jl_timing_suspend_ctor(&__timing_suspend, #subsystem, ct)
 #endif
 
+// Counting
+#ifdef USE_ITTAPI
+#define _ITTAPI_COUNTER_MEMBER __itt_counter ittapi_counter;
+#else
+#define _ITTAPI_COUNTER_MEMBER
+#endif
+
+#ifdef USE_TIMING_COUNTS
+#define _COUNTS_MEMBER _Atomic(uint64_t) basic_counter;
```

Comment:
Why is this unsigned?

Reply:
No real reason, it could be signed but I don't know of a useful statistic that is signed at the moment.

Reply:
This is now fixed by just printing everything out as a signed integer rather than unsigned.